### ISSUE TITLE
Add support for Venu 3 / venu 3 s

### ISSUE THREAD

**Issue by pedrorijo91** [pedrorijo91/garmin-padel#84]
venu 3

* https://app.hey.com/topics/1232218915

venu 3s

* https://app.hey.com/topics/1375085963

device has 3 right buttons, but no left buttons

**Comment by xpresas** [comment]
I would be willing to test if needed!

**Comment by pedrorijo91** [comment]
thank you @xpresas . not sure if I will work on it anytime soon tbh

**Comment by xpresas** [comment]
Ehh I might try to copy ur project and rework it then. Althou monkey C is far from the language I use... :) Thanks anyways

**Comment by pedrorijo91** [comment]
yeah, monkey C is weird 😅  if you want to submit a PR to add support for touchscreen devices I'm happy to review, merge, and release it. and to provide some guidance whenever i can

**Comment by pedrorijo91** [comment]
(that's why i added the project as open source, so that anyone can use it, and help adding support so that we can all benefit from it :D )
